pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -473,8 +473,6 @@ ASTDeclarativeUnit DeclarativeUnit()  :
 		ProgramUnit()  |
 		LOOKAHEAD(4)
 		VariableOrConstantDeclaration() |
-		LOOKAHEAD(2)
-		CursorSpecification() |
```

Comment:
Removing CursorSpecification actually breaks compatibility, because now the Cursor is parsed as CursorBody

Reply:
You're right, thanks.